alvr/server/src/openvr/props.rs

```diff
@@ -115,7 +120,7 @@ pub extern "C" fn set_device_openvr_props(device_id: u64) {
         fn set_prop(prop: OpenvrProperty) {
             info!("Setting head OpenVR prop: {prop:?}");
             unsafe {
-                crate::SetOpenvrProperty(*HEAD_ID, to_ffi_openvr_prop(prop));
+                crate::openvr::SetOpenvrProperty(*HEAD_ID, to_ffi_openvr_prop(prop));
```

Comment:
Just import it too

Reply:
I was a bit sloppy with imports, but i will redo them when splitting the crate in two.